pkg/network/usm/monitor_test.go

```diff
@@ -668,17 +668,38 @@ func cleanProtocolMaps(t *testing.T, protocolName string, manager *manager.Manag
 		_, shouldOnlyZero := mapTypesToZero[mapInstance.Type()]
 
 		key := make([]byte, mapInstance.KeySize())
-		value := make([]byte, mapInstance.ValueSize())
+
+		var value any
+		var emptyValue any
+
+		if mapInstance.Type() == ebpf.PerCPUArray {
+			possibleCPUs, err := ebpf.PossibleCPU()
+			if err != nil {
+				panic("failed to get possible CPUs")
+			}
+			value = make([][]byte, possibleCPUs)
+			for i := range value.([][]byte) {
+				value.([][]byte)[i] = make([]byte, mapInstance.ValueSize())
+			}
+
+			emptyValue = make([][]byte, possibleCPUs)
+			for i := range emptyValue.([][]byte) {
+				emptyValue.([][]byte)[i] = make([]byte, mapInstance.ValueSize())
+			}
```

Comment:
the readability of the code is not good enough (relates to my other [comment](https://github.com/DataDog/datadog-agent/pull/30657/files#r1825050747))
splitting the code into case will make it look better

Reply:
```suggestion
			value = make([][]byte, possibleCPUs)
			emptyValue = make([][]byte, possibleCPUs)
			for i := 0; i < possibleCPUs; i++  {
				value.([][]byte)[i] = make([]byte, mapInstance.ValueSize())
				emptyValue.([][]byte)[i] = make([]byte, mapInstance.ValueSize())
			}
```